testing/compile-tracee.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
+
+#
+# This script downloads and updates the aquasecurity/tracee repository
+# 
+
+die() {
+    echo ${@}
+    exit 1
+}
+
+BASEDIR=$(dirname "${0}") ; cd ${BASEDIR}/../ ; BASEDIR=$(pwd) ; cd ${BASEDIR}
+TRACEE_REPO="https://github.com/aquasecurity/tracee.git"
+TRACEE_DIR="${BASEDIR}/testing/tracee"
+RETURN_CODE=0
+
+CMDS="rsync git cp rm mv"
+for cmd in ${CMDS}; do
+    command -v $cmd 2>&1 >/dev/null || die "cmd ${cmd} not found"
+done
+
+branch_clean() {
+    cd ${1} || die "could not change dirs"
```

Comment:
```
++ basename /home/rafaeldtinoco/work/ebpf/libbpfgo/testing/tracee
+ die 'tracee not a repo dir'
+ echo tracee not a repo dir
tracee not a repo dir
```

This is not right, you are CWDing to ./testing/tracee and it does not contain Readme.md file.

Reply:
I don't understand what you mean, doesn't it?

Reply:
Nm, my local tree had garbage, sorry.